client/src/panels/panel_declarations/results/result_drilldown/result_displays.js

```diff
@@ -95,8 +95,8 @@ export const ResultCounts = ({ base_hierarchy, doc, subject }) => {
 
           doc_year: result_docs[doc].year,
 
-          num_programs: count_items.program,
-          num_prog_results: count_items.result,
+          num_programs: count_items.program || 0,
+          num_prog_results: count_items.result || 0,
           num_results: (count_items.result || 0) + (count_items.dr || 0),
           num_indicators: count_items.indicator,
```

Comment:
For total consistency, should `num_indicators` default to 0 too?

Reply:
I totally meant to do that, just was so focused on closing it quickly my eyes skimmed right over it 🤣